ansible/vars/enikshay/enikshay_public.yml

```diff
@@ -0,0 +1,289 @@
+old_s3_blob_db_access_key: "{{ secrets.OLD_S3_BLOB_DB_ACCESS_KEY }}"
+old_s3_blob_db_secret_key: "{{ secrets.OLD_S3_BLOB_DB_SECRET_KEY }}"
+
+proxy_type: 'nginx'
+
+primary_ssl_env: "enikshay"
+
+ansible_user_password_sha_512: "{{ secrets.ANSIBLE_USER_PASSWORD_SHA_512 }}"
+fake_ssl_cert: no
+
+NEW_RELIC_KEY: "{{ secrets.NEW_RELIC_KEY }}"
+run_newrelic_plugin_agent: False
+
+DATADOG_ENABLED: True
+DATADOG_API_KEY: "{{ secrets.DATADOG_API_KEY }}"
+DATADOG_APP_KEY: "{{ secrets.DATADOG_APP_KEY }}"
+datadog_integration_cloudant: False
+datadog_integration_vmware: True
+
+elasticsearch_endpoint: '{{ groups.elasticsearch.0 }}'
+elasticsearch_memory: '8192m'
+elasticsearch_cluster_name: 'enikshayes-1.x'
+
+apache2_cert_file: '../config/{{ deploy_env }}/ssl/enikshay.in.crt'
+apache2_key_file: '../config/{{ deploy_env }}/ssl/enikshay.in.key'
+apache2_ca_file: '../config/{{ deploy_env }}/ssl/intermediate.crt'
+# Nginx only accepts on cert, so cat the site cert and chain:
+# $ cat india.commcarehq.org.crt intermediate.crt > india.commcarehq.org.combined.crt
+nginx_combined_cert_file: '../config/{{ deploy_env }}/ssl/enikshay.in.combined.crt'
+
+
+#tableau_nginx_combined_cert_file: '../config/{{ deploy_env }}/ssl/icds.commcarehq.org.combined.crt'
+#tableau_key_file: '../config/{{ deploy_env }}/ssl/icds.commcarehq.org.key'
+
+#cas_nginx_combined_cert_file: '../config/{{ deploy_env}}/ssl/cas.commcarehq.org.combined.crt'
+#cas_key_file: '../config/{{ deploy_env }}/ssl/cas.commcarehq.org.key'
+
+enikshay_nginx_combined_cert_file: '../config/{{ deploy_env}}/ssl/enikshay.commcarehq.org.combined.crt'
+enikshay_key_file: '../config/{{ deploy_env }}/ssl/enikshay.commcarehq.org.key'
+
+formplayer_sentry_dsn: '{{ secrets.FORMPLAYER_SENTRY_DSN }}'
+
+supervisor_http_enabled: True
+supervisor_http_username: "{{ secrets.SUPERVISOR_HTTP_USERNAME }}"
+supervisor_http_password: "{{ secrets.SUPERVISOR_HTTP_PASSWORD }}"
+
+keystore_file: '../config/DimagiKeyStore'
+
+backup_blobdb: False
+backup_postgres: True
+backup_es: False
+postgres_s3: False
+backup_couch: True
+nadir_hour: 16
+
+#aws_region: 'ap-south-1'
+
+nofile_limit: 65536
+
+postgres_users: "{{ secrets.POSTGRES_USERS }}"
+
+postgresql_max_connections: 300
+pgbouncer_max_connections: 800
+pgbouncer_default_pool: 290
+pgbouncer_reserve_pool: 5
+pgbouncer_pool_timeout: 1
+pgbouncer_pool_mode: transaction
+
+formplayer_db_name: formplayer
+
+postgresql_dbs:
+  - django_alias: default
+    name: "{{localsettings.PG_DATABASE_NAME}}"
+#  - django_alias: proxy
+#    name: commcarehq_proxy
+#  - django_alias: p1
+#    name: commcarehq_p1
+#    shards: [0, 204]
+#  - django_alias: p2
+#    name: commcarehq_p2
+#    shards: [205, 409]
+#  - django_alias: p3
+#    name: commcarehq_p3
+#    shards: [410, 614]
+#  - django_alias: p4
+#    name: commcarehq_p4
+#    shards: [615, 819]
+#  - django_alias: p5
+#    name: commcarehq_p5
+#    shards: [820, 1023]
+  - name: "{{localsettings.UCR_DATABASE_NAME}}"
```

Comment:
looked at this again and I see it's not necessary for this to have a django alias for this.